specification/securityinsights/resource-manager/Microsoft.SecurityInsights/preview/2019-01-01-preview/SecurityInsights.json

```diff
@@ -3562,6 +3562,467 @@
           }
         }
       }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/{operationalInsightsResourceProvider}/workspaces/{workspaceName}/providers/Microsoft.SecurityInsights/threatIntelligence/main/createIndicator": {
```

Comment:
>createIndicator [](start = 208, length = 15)

Note for other reviewers: Discussed this at the meeting. There is no natural ID that the user can provide and the ID is backend generated. I'm approving the creation of a resource via POST.